packages/store/addon/-private/ts-interfaces/record-data-store-wrapper.ts

```diff
@@ -1,15 +1,35 @@
-import { RelationshipsSchema, AttributesSchema } from "./record-data-schemas";
+import { RelationshipsSchema, AttributesSchema } from './record-data-schemas';
+import Store from '../system/store';
+
+type Store = InstanceType<typeof Store>;
+
 export interface RecordDataStoreWrapper {
+  store: Store;
```

Comment:
if we do this shouldn't this be underscored?

Reply:
it was already present, this just updated the interface, but after oob discussion moved to underscore